components/ActionMenu/ActionMenu.js

```diff
@@ -0,0 +1,102 @@
+import {
+  materialContact,
+  materialMeeting,
+  materialPlus,
+} from "@/assets/Icons8";
+import Image from "next/image";
+import {
+  Dropdown,
+  DropdownItem,
+  StyledActionMenu,
+  StyledImage,
+  StyledLink,
+  StyledSpan,
+} from "./ActionMenu.styled";
+import { useEffect, useRef, useState } from "react";
+import { Tooltip } from "react-tooltip";
+
+export default function ActionMenu() {
+  const [dropdown, setDropdown] = useState(false);
+
+  function closeDropdown() {
+    dropdown && setDropdown(false);
+  }
+
+  const actionMenuItems = [
+    {
+      icon: materialContact,
+      iconAlt: "Contact icon",
+      title: "New contact",
+      url: "/new",
+    },
+    {
+      icon: materialMeeting,
+      iconAlt: "Interaction icon",
+      title: "New interaction",
+      url: "/interactions/new",
+    },
+  ];
+
+  let ref = useRef();
```

Comment:
Why do you use let here? Will you reassign it later?

Reply:
Thx, I changed it to const.